ts/common.ts

```diff
@@ -1,9 +1,25 @@
 import { OrderBookApi, SupportedChainId } from "@cowprotocol/cow-sdk";
-import { BigNumber, ethers } from "ethers";
+import { ethers } from "ethers";
 import { multicall3Abi } from "./abi";
 import readline from "readline";
 
-export const networkSpecificConfigs = {
+interface NetworkDetails {
+  rpcUrl: string;
+  explorer: string;
+}
+
+export const SUPPORTED_NETWORKS = [
+  "mainnet",
+  "gnosis",
+  "arbitrum",
+  "base",
+  "sepolia",
+] as const;
```

Comment:
nit: if it adapts well you have this too https://github.com/cowprotocol/cow-sdk/blob/63c2f1608d660c1b2cb2e3544b5669d5ec4408a1/src/chains/const/index.ts#L29 and https://github.com/cowprotocol/cow-sdk/blob/63c2f1608d660c1b2cb2e3544b5669d5ec4408a1/src/chains/const/index.ts#L16

Reply:
feel free to ignore if this adds any work, as this was not part of the scope of this change, and you just refactored what was there

Reply:
We have a RC on 6.0.0 that should have it. 
Anyways, is not important for this project. Maybe when we add our next network we can add it https://github.com/cowprotocol/cow-sdk/blob/5208521e2960edc408746c13ebc3c3ea2f271700/src/chains/types.ts#L10